ui/components/MesheryApplications.js

```diff
@@ -243,6 +248,40 @@ function MesheryApplications({
     );
   };
 
+  const handleUnDeploy = (application_file) => {
+    updateProgress({ showProgress : true })
+    dataFetch(
+      "/api/pattern",
+      {
+        credentials : "include",
+        method : "DELETE",
+        body : JSON.stringify({ k8s_manifest : application_file }),
```

Comment:
How you are making sure that the application file provided is a k8s_manifest, in case it is not then the whole undedeploy operation will fail.

Reply:
This should solve for the K8scontext....?

Reply:
added in suggested change.

Reply:
Opps I missed it, you said for K8s_manifest